packages/core/types/index.d.ts

```diff
@@ -241,8 +241,8 @@ export interface RunSagaOptions<A, S> {
 }
 
 export const CANCEL: string
-export const END: END
-export type END = END
+export const END: EndType
```

Comment:
was this change needed? 🤔

Reply:
I'm not sure. But this is the only way I could fix [strange build error](https://travis-ci.org/redux-saga/redux-saga/jobs/592087832#L720).